### PR TITLE
FF-130 Architecture Wiki

### PR DESCRIPTION
**Pull request by open-schnick**
@Gimleux can you draw a bit around the class diagram, to group the classes in Controller / Logic, Models / Entities, View Models / DTO's?

### CODE REVIEW

File: wiki/SoftwareArchitectureDocument.md [hunk 1]
```diff
@@ -79,39 +77,44 @@ As mentioned the backend is using the MVC pattern. This enables a clean software
...
-tdb @open-schnick
 The logical view for our application follows the Spring Boot architecture and looks like:
 
-In our specific case the view however is not part of spring but provided separately as an webapp front end.
+<figure>
+  <img src="/assets/images/arch-doc/spring_boot_logical_view.png"  alt=""/>
+</figure>
 
+In our specific case the view however is not part of spring but provided separately as a web application front end.
+The front end application handles all the user interaction and independently handles the view coordination thus fulfilling the roles of view and dispatcher alike. However view and dispatcher do not interact with the client independently instead the dispatcher has been substituted by the ViewModel which connects the view and the model as describe above as well as forming the connection to the controller.
```
Comment (qvalentin) [inline wiki/SoftwareArchitectureDocument.md:106]: Described

**Review by qvalentin — APPROVED** [review]

**Comment by qvalentin** [comment]
# TODO:
- copy stuff from the blogpost (text + **diagrams**)

**Comment by open-schnick** [comment]
> # TODO:
> 
> - copy stuff from the blogpost (text + **diagrams**)
> 
> 

Wdym?

**Comment by qvalentin** [comment]
> > # TODO:
> > 
> > * copy stuff from the blogpost (text + **diagrams**)
> 
> Wdym?

We already got some good text and diagramm here:
https://filefighter.github.io/tech/2020/10/10/architecture.html

Why not just copy some of it?

**Comment by qvalentin** [comment]
nice

**Pull request merged**